content/src/controller.js

```diff
@@ -358,27 +358,34 @@ view.on('share', function() {
       console.log("Nothing to share.");
       return;
     } else if (doc.data !== '') { // If program is not empty, generate thumbnail.
-      var iframe = document.getElementById('output-frame');
-      thumbnailDataUrl = thumbnail.generateThumbnailDataUrl(iframe);
-    }
-    var data = $.extend({
-      thumbnail: thumbnailDataUrl
-    }, modelatpos('left').data, doc);
-    storage.saveFile('share', sharename, data, true, 828, false, function(m) {
-      var opts = { title: shortfilename };
-      if (!m.error && !m.deleted) {
-        opts.shareStageURL = "//share." + window.pencilcode.domain +
-            "/home/" + sharename;
-      }
-      if (model.ownername) {
-        // Share the run URL unless there is no owner (e.g., for /first).
-        opts.shareRunURL = "//" + document.domain + '/home/' +
-            modelatpos('left').filename;
+      if (model.tempThumbnail) {
```

Comment:
What had to change here to fix share-site thumbnail generation?  Can you add a short comment to explain?